lib/teaspoon/suite.rb

```diff
@@ -53,7 +53,16 @@ def asset_tree(sources)
         asset = @env.find_asset(source)
 
         if asset && asset.respond_to?(:logical_path)
-          assets = config.expand_assets ? asset.to_a : [asset]
+          assets =
```

Comment:
Might want to move this to a different method that can be easily overridden if the case comes up.